src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/19.diff.sql

```diff
@@ -0,0 +1,302 @@
+﻿/*************************************************************
+    Create partition function, scheme, and clustered index to migrate data on resource change data table.
+**************************************************************/
+
+IF NOT EXISTS(SELECT * FROM sys.partition_functions WHERE name = 'PartitionFunction_ResourceChangeData_Timestamp')
+BEGIN 
+    -- Partition function for the ResourceChangeData table.
+    -- It is not a fixed-sized partition. It is a sliding window partition.
+    -- Adding a range right partition function on a timestamp column. 
+    -- Range right means that the actual boundary value belongs to its right partition,
+    -- it is the first value in the right partition.
+    CREATE PARTITION FUNCTION PartitionFunction_ResourceChangeData_Timestamp (datetime2(7))
```

Comment:
As discussed, confirm with Katherine that `datetime2` boundary values won't cause inconsistencies with different session languages (see the [note](https://docs.microsoft.com/en-us/sql/t-sql/statements/create-partition-function-transact-sql?view=sql-server-ver15#arguments) in this documentation).